tests/integration.rs

```diff
@@ -6229,6 +6229,13 @@ mod common_parallel {
             &event_path_restored
         ));
 
+        // Remove the snapshot dir
+        Command::new("rm")
+            .arg("-rf")
+            .arg(snapshot_dir.as_str())
+            .output()
+            .unwrap();
+
```

Comment:
Please use this instead: https://doc.rust-lang.org/std/fs/fn.remove_dir_all.html

Reply:
Done.